contracts/token/IZNSDomainToken.sol

```diff
@@ -55,11 +58,15 @@ interface IZNSDomainToken is IERC2981, IERC721 {
 
     function setDefaultRoyalty(address receiver, uint96 royaltyFraction) external;
 
+    function transferTokenFrom(address from, address to, uint256 tokenId) external;
```

Comment:
maybe make a more clear name?...
idk... "changeTokenOwner()` or something... "transferTokenFrom" is kind of a good name, but may be a bit confusing?

Reply:
I felt making it as close to the regular `transferFrom` was best, so `transferTokenFrom` made sense because it was transferring *just* the token, but maybe `setTokenOwner` or `updateTokenOwner` to match verbiage from the registry would be better